_rest_routes.inc.php

```diff
@@ -571,6 +572,18 @@
         $return = (new PrescriptionRestController())->getOne($uuid);
         RestConfig::apiLog($return);
         return $return;
+    },
+    "GET /api/surgery" => function ($uuid) {
+        RestConfig::authorization_check("patients", "med");
+        $return = (new SurgeryRestController())->getAll();
+        RestConfig::apiLog($return);
+        return $return;
+    },
+    "GET /api/surgery/:uuid" => function ($uuid) {
+        RestConfig::authorization_check("patients", "med");
+        $return = (new SurgeryRestController())->getOne($uuid);
+        RestConfig::apiLog($return);
+        return $return;
```

Comment:
I would drop  the changes in this script and drop the src/RestControllers/SurgeryRestController.php  script since there is already a mechanism in non-fhir api to collect surgeries.
(but still keep all the fhir related code that you have)

Reply:
@shubham17061999 , if not using, would remove all changes in the _rest_routes.inc.php script and would completely remove the src/RestControllers/SurgeryRestController.php script

Reply:
Missed this. Sorry for the delay.